Doc/library/compression.zstd.rst

```diff
@@ -0,0 +1,780 @@
+:mod:`!compression.zstd` --- Compression compatible with the Zstandard format
+=============================================================================
+
+.. module:: compression.zstd
+   :synopsis: Low level interface to compression and decompression routines in
+              Meta's zstd library
```

Comment:
Wouldn't call out Meta here. The reference manual for the zstd itself doesn't mention a company name.  On the other hand, it might not be a bad idea to reference that manual somewhere (https://facebook.github.io/zstd/zstd_manual.html).

Reply:
Hm, the library is called `zstd`, perhaps we should say "in the zstd library"?

Reply:
I second the suggestion to link to the [zstd manual](https://facebook.github.io/zstd/zstd_manual.html), and to copy or quote the short explanation: "... is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level and better compression ratios."

Reply:
I ended up opening https://github.com/facebook/zstd/issues/4388 because it turns out the SEO first result is not the latest version linked from https://facebook.github.io/zstd/, which is version specific.

For now I think I'll leave out a link to the manual and add one later once that issue is resolved.

I've removed the Meta reference and said "in the zstd library."

Reply:
Yann was quick! That issue is resolved and I just pushed up the requested changes.